src/Classes/FrameManager.ts

```diff
@@ -22,6 +22,8 @@ export class FrameManager {
 
         const delta = timestamp - container.lastFrameTime;
 
+        console.log(delta, container.lastFrameTime, timestamp);
```

Comment:
Can we remove this console.log? It flooding my production environment 😅

Reply:
Ops, yeah sure I'm releasing a 1.13.8 for another bug I found